scripts/new-upstream-snapshot

```diff
@@ -459,17 +406,8 @@ main() {
     fi
 
     local clog="${TEMP_D}/changelog" gitlog="${TEMP_D}/gitlog"
-    git log --first-parent --no-decorate --format=full \
-        "${prev_pkg_hash}..${from_ref}" >  "$gitlog" ||
-        fail "failed git log ${prev_pkg_hash}..${from_ref}"
-
-    local entries=""
-    header="  * $new_msg${sru_bug_string:+ ${sru_bug_string}}"
-    entries=`cat $gitlog | log2dch $skip_bugs`
-    add_changelog_entries "$header\n$entries" "${new_pkg_ver}" || fail
-
-    cat "$gitlog" | log2dch $skip_bugs > "new-upstream-changes.txt" ||
-        fail "failed log2dch"
+    header="$new_msg${sru_bug_string:+ ${sru_bug_string}}"
+    python3 $(dirname $0)/add_changelog.py "${header}" "${new_pkg_ver}" "${bug_refs_in_clog}" || fail
```

Comment:
```suggestion
    python3 "$(dirname "$0")"/add_changelog.py "${header}" "${new_pkg_ver}" "${bug_refs_in_clog}" || fail
```
To prevent word splitting and globbing.